src/output.rs

```diff
@@ -78,10 +93,14 @@ impl OutputType {
                     let mut p = Command::new(&pager_path);
                     if args.is_empty() || replace_arguments_to_less {
                         p.arg("--RAW-CONTROL-CHARS");
-                        if quit_if_one_screen {
+                        if let SingleScreenAction::Quit = single_screen_action {
```

Comment:
Using `single_screen_action == SingleScreenAction::Quit` would be a bit clearer here.. but maybe that's just me. It would require deriving `PartialEq`.

Similar below.

Reply:
Initially I used PartialEq , but then switched to the `if let` in order to skip yet another derive which slows down compile time. Nevertheless, I've update the code with the suggestion.